build.gradle.kts

```diff
@@ -49,8 +49,8 @@ dependencies {
 
 tasks.withType<KotlinCompile>().configureEach {
    kotlinOptions.jvmTarget = "1.8"
-   kotlinOptions.apiVersion = "1.5"
-   kotlinOptions.languageVersion = "1.5"   
+   kotlinOptions.apiVersion = "1.9"
```

Comment:
Is it required to increase the apiVersion? It requires all of our users to update their codebase to kotlin 1.9. This seems to be a big change for a small cosmetic API change.

Reply:
Now we don't want the `@Repeatable`. But if we want it for another annotation, then we will need kotlin 1.9. Previous versions has been tested and we can only see the last (or first) repeated annotation

Reply:
And also, keeping a `-4` version is the open door to migration surprises, maybe we should keep a support to `-1` or `-2` versions no ? By the way I'm not totally comfortable about kotlinVersion (libraries) vs apiVersion vs languageVersion, what are the real impacts on users. I'll take a look

Reply:
And also, currently I'm working with the latest version of kotlinx serialization that is on kotlin 1.8 or 1.9. If we want a full power library, then we will need to migrate